docs/source/rpc.rst

```diff
@@ -0,0 +1,42 @@
+.. role:: hidden
+    :class: hidden-section
+
+Distributed RPC Framework
+=====================================================
+
+The distributed RPC framework provides mechanisms for multi-machine model training through a set of primitives to allow for remote communication, and a higher-level API to automatically differentiate models split across several machines.
```

Comment:
Why do we have everything on a single long line? Can we split the lines in this file? Looks like most doc files split lines after 80 chars.